src/wavesurfer.js

```diff
@@ -1451,9 +1452,11 @@ export default class WaveSurfer extends util.Observer {
             this.backend.once('error', err => this.fireEvent('error', err))
         );
 
+        // If peaks are provided, render them and fire the `waveform-ready` event.
```

Comment:
also, method needs to have an `@emits` for the event

Reply:
Added.